### ISSUE TITLE
Write formatted JSON to NCF file

### ISSUE THREAD

**Issue by marcvanbreemen** [jenius-apps/nightingale-rest-api-client#257]
Currently the content of the NCF isn't a formatted JSON.

With a formatted JSON (line breaks, indented, etc) it would increase the readability especially if I want to include in my project's Git VC.

**Comment by dpaulino** [comment]
You're referring to the exported NCF? I could provide an option to format it. The idea with leaving it unformatted was to reduce storage usage, but I can look into this

**Comment by marcvanbreemen** [comment]
No, not the export. I use the export only once to create a NCF in my project's repo folder. After that I edit it from there.

What I was thinking of is that the NCF file always saves in a formatted way.

Your remark about saving storage was reminding me of the time I was using "space removers" for GWBasic code, so that I could squeeze as much code as possible in the huge 22 Kb memory I had available. .... ;). These days I can handle a few Kb of storage just fine :).
